projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -92,17 +92,23 @@ export class ExplorerComponent {
   public readonly resultsDashboard$: Observable<ExplorerGeneratedDashboard>;
   public readonly vizDashboard$: Observable<ExplorerGeneratedDashboard>;
 
-  public contextItems: ToggleItem<ContextLabel>[] = [
+  public contextItems: ToggleItem<ExplorerContextScope>[] = [
     {
       label: ExplorerComponent.API_TRACES,
-      value: ExplorerComponent.API_TRACES
+      value: {
+        context: ObservabilityTraceType.Api,
+        scope: ScopeQueryParam.EndpointTraces
```

Comment:
scope is a bit of an overloaded term (context actually replaces scope for explorer, which is lovely) - what we're putting in this scope field is actually a query parameter that's local to this component. All the usage is right, just naming question. Maybe just match the enum with `scopeQueryParam` ?